dashboard/app/controllers/projects_controller.rb

```diff
@@ -696,4 +738,50 @@ def extra_links
       view_options(responsive_content: true)
     end
   end
+
+  # Temporary - will be replaced with storing in database.
+  private def send_project_submission(name, username, project_type, channel_id, description)
+    unless Rails.env.development? || Rails.env.test?
+      subject = 'TESTING: Featured project gallery submission'
+      response = HTTParty.post(
+        'https://codeorg.zendesk.com/api/v2/tickets.json',
+        headers: {"Content-Type" => "application/json", "Accept" => "application/json"},
+        body: {
+          ticket: {
+            requester: {
+              name: username
```

Comment:
![Screenshot 2024-10-31 at 11 59 06 AM](https://github.com/user-attachments/assets/448864aa-633f-46a8-bff3-ab8761687585)

Reply:
I tried sending the Zendesk ticket without a requestor, and interestingly, the request was assigned 'Eng. Team (used w/automations). 
<img width="1012" alt="Screenshot 2024-10-31 at 12 32 09 PM" src="https://github.com/user-attachments/assets/e19e333b-c826-44ca-ba3c-ca0d23255b85">

If an email address is not included, the Zendesk ticket is still sent successfully. However, an existing Zendesk user account is not updated with this ticket. This seems okay since submitting submissions via Zendesk is temporary.